src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -961,3 +962,22 @@ def is_from_cli():
     from promptflow._core.operation_context import OperationContext
 
     return CLI_UA in OperationContext.get_instance().get_user_agent()
+
+
+def is_url(value: Union[PathLike, str]) -> bool:
+    try:
+        result = urlparse(str(value))
+        return all([result.scheme, result.netloc])
+    except ValueError:
+        return False
+
+
+def is_remote_uri(obj) -> bool:
```

Comment:
may have conflict with this PR: https://github.com/microsoft/promptflow/pull/1226/files

Reply:
This is an easy merge. I can wait and merge.